src/pudl/transform/ferc1.py

```diff
@@ -717,63 +715,69 @@ def drop_null_data_rows(self, df: pd.DataFrame) -> pd.DataFrame:
         return df.drop(index=probably_totals_index)
 
 
-########################################################################################
-########################################################################################
-# Christina's Transformer Classes
-########################################################################################
-########################################################################################
-class PlantsSteamFerc1:
+class PlantsSteamFerc1TableTransformer(Ferc1AbstractTableTransformer):
     """Transformer class for the plants_steam_ferc1 table."""
 
-    def execute(
+    table_id: Ferc1TableId = Ferc1TableId.PLANTS_STEAM_FERC1
+
+    def transform(
         self,
         raw_dbf: pd.DataFrame,
         raw_xbrl_instant: pd.DataFrame,
         raw_xbrl_duration: pd.DataFrame,
         transformed_fuel: pd.DataFrame,
     ):
         """Perform table transformations for the plants_steam_ferc1 table."""
-        plants_steam_combo = (
-            self.pre_concat_clean_and_concat_dbf_xbrl(
-                raw_dbf, raw_xbrl_instant, raw_xbrl_duration
-            )
-            .pipe(self.simplify_strings_for_table)
-            .pipe(self.clean_strings_for_table)
-            .pipe(self.oob_to_nan_for_table)
-            .pipe(self.convert_units_in_table)
-            .pipe(self.convert_table_dtypes)
-            .pipe(plants_steam_assign_plant_ids, ferc1_fuel_df=transformed_fuel)
-        )
-        plants_steam_validate_ids(plants_steam_combo)
-        return plants_steam_combo
-
-    def process_dbf(self, raw_dbf):
-        """Modifications of the dbf plants_steam_ferc1 table before concat w/ xbrl."""
-        plants_steam_dbf = self.rename_columns(
-            raw_table=raw_dbf,
-            source="dbf",
-        ).pipe(self.assign_record_id, source_ferc1=Ferc1Source.DBF)
-        return plants_steam_dbf
-
-    def process_xbrl(self, raw_xbrl_instant, raw_xbrl_duration):
-        """Modifications of the xbrl plants_steam_ferc1 table before concat w/ xbrl."""
-        plants_steam_xbrl = (
-            self.merge_instant_and_duration_tables(
-                duration=raw_xbrl_duration,
-                instant=raw_xbrl_instant,
+        self.plants_steam = (
```

Comment:
this is a good question! I've personally been using this to test the very time consuming plant-id-er. when it fails, its often because of the input df so its useful to save that df to the class so you can poke at it.

deleting it after the plant-id-er/before `return` would be a good solution.

I don't reaaaally think this should be a thing we do for other tables. Maybe this is just because the incremental updates (removing the chaining) would be much less compact/nice to read. If we go in this direction, I would rather the transform methods themselves all save to `self._tmp_df`. But really unless there is another time consuming transform, I don't think this is necessary.

Reply:
I decided to just remove this for now because I want to move forward and do not want to wait to set something up that is more complex. It's not that hard to go in and comment out the time consuming task.